packages/button/src/button.tsx

```diff
@@ -4,14 +4,30 @@ import { pxToPt } from "./utils/px-to-pt";
 
 export type ButtonProps = Readonly<React.ComponentPropsWithoutRef<"a">>;
 
+const maxFontWidth = 5;
+
+/**
+ * Computes a msoFontWidth \<= 5 and a count of space characters that,
+ * when applied, end up being as close to `expectedWidth` as possible.
+ */
 function computeFontWidthAndSpaceCount(expectedWidth: number) {
+  if (expectedWidth === 0) return [0, 0];
```

Comment:
can we use explicit blocks? 
```suggestion
  if (expectedWidth === 0) {
    return [0, 0];
  }
```